src/vm/functions/mod.rs

```diff
@@ -367,3 +369,31 @@ fn special_as_contract(args: &[SymbolicExpression], env: &mut Environment, conte
 
     result
 }
+
+fn special_contract_of(args: &[SymbolicExpression], env: &mut Environment, context: &LocalContext) -> Result<Value> {
+    // (contract-of (..))
+    // arg0 => trait
+    check_argument_count(1, args)?;
+
+    runtime_cost!(cost_functions::CONTRACT_OF, env, 0)?;
+
+    let contract_ref = match &args[0].expr {
+        SymbolicExpressionType::Atom(_contract_ref) => {
+            _contract_ref
```

Comment:
Linting -- the variable `_contract_ref` should not need a `_` prefix if it is going to be used.  Same goes for `_contract_identifier` and `_trait_identifier` below.

Reply:
indeed, `_` not needed `contrac_ref` and `contract_identifier`.  Eagle eyes here
However, `_trait_identifier` is not used anywhere and needs it.